config/database.go

```diff
@@ -49,3 +50,11 @@ func DbURL(dbConfig *DBConfig) string {
 		dbConfig.DBName,
 	)
 }
+
+func MysqlConn() (*gorm.DB, error) {
+	db, err := gorm.Open(mysql.Open(DbURL(BuildDBConfig())), &gorm.Config{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	return db, err
+}
```

Comment:
I don't think its good practice to put our db reference into a function, cause it'll open a db connection everytime we need to access our model. Or maybe you have your own opinion about it.

Reply:
well, because of gorm in latest version doesn't require `db.Close()` anymore, i decided to move connections to config DB. Even jinzu has explained why he removed `db.Closed()` [here](http://disq.us/p/2ayeg92). Also, in docs he has [shown](https://gorm.io/docs/#Quick-Start) that it's enough to use error handling to control db connection. Quite easy setup in config and you can see in my latest commit

Reply:
Does the program open up a new connection everytime we called the function?

Reply:
Well, jinzu hasn't a clear statement about this. Even in the issues, he seems like ingnoring this. I don't really know why :/

Reply:
Well it's fine run, just want some explanation.